project proposal.ipynb

```diff
@@ -77,10 +77,10 @@
     "We almost lost hope - but eventually we managed to find two interesting sources of data. \n",
```

Comment:
<p>Normalization window isnt for the past 10 days - but for the past/previews 10 rows - were each row represent 10 minutes of aggregated data.</p><blockquote>every point in time will be normalized acourding to the past <s>10 days</s> <strong>10 data points / 10 rows of 10 minutes each</strong></blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/alongouldman/stock_market_neural_nets/pull/9/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>